democracy_club/templates/polling-stations/polling_stations_base.html

```diff
@@ -9,7 +10,7 @@
       <ul>
         <li {% if request.path == "/projects/polling-stations/" %}class="selected"{% endif %}><a href="{% url "polling_one_pager" %}">Overview</a></li>
         <li {% if request.path == "/projects/polling-stations/embed/" %}class="selected"{% endif %}><a href="{% url "polling_embed_code" %}">Use on your website</a></li>
-        <li {% if request.path == "/projects/polling-stations/upload/" %}class="selected"{% endif %}><a href="{% url "polling_data_upload" %}">How to send us data</a></li>
+        {# <li {% if request.path == "/projects/polling-stations/upload/" %}class="selected"{% endif %}><a href="{% url "polling_data_upload" %}">How to send us data</a></li> #}
```

Comment:
I think its kind of weird that there is no menu item for this. If you're on this page you're in a sort of navigational no-man's land but I can see you've consciously removed it. Is that so you have to read the pricing page to find the link?

Reply:
Yeah, see the comment in this commit for my reckons: https://github.com/DemocracyClub/Website/pull/118/commits/946e830f24317953b45d0f5fa9670510a910de26

Reply:
Oh right, sorry, yeah. I only looked at the staging site - didn't read the commit messages